### DISCUSSION TITLE
pydantic version requirement clashes with that of vLLM

### DISCUSSION THREAD

**Discussion by mhillebrand** [outlines-dev/outlines#491] [Q&A] [answered]
Outlines is supposed to offer vLLM integration now. However, they don't seem to play nicely with each other:

```
$ pip install -U outlines
outlines 0.0.19 has requirement pydantic>=2.0, but you have pydantic 1.10.13.

$ pip install -U pydantic
vllm 0.2.6 requires pydantic==1.10.13, but you have pydantic 2.5.3 which is incompatible.
```

**Comment by rlouf** [comment, marked as answer]
If you re-install pydantic>=2.0 after installing vLLM this should work. Not ideal, I know.

**Reply by rlouf** [reply]
This is tracked in https://github.com/outlines-dev/outlines/issues/504. vLLM is currently considering bumping the Pydantic version and it shouldn't be an issue soon.

**Reply by rlouf** [reply]
This is now solved since vLLM released `0.3.0`
